src/navigations/root-stack.js

```diff
@@ -152,182 +160,182 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="DetailGroupImage"
-          component={DetailGroupImage}
+          component={withKeyboardWrapper(DetailGroupImage)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="ChatDetailPage"
-          component={ChatDetailPage}
+          component={withKeyboardWrapper(ChatDetailPage)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="ReplyComment"
-          component={ReplyComment}
+          component={withKeyboardWrapper(ReplyComment)}
           options={{headerShown: false}}
         />
 
         <AuthenticatedStack.Screen
           name="ProfileReplyComment"
-          component={ProfileReplyComment}
+          component={withKeyboardWrapper(ProfileReplyComment)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="OtherProfileReplyComment"
-          component={OtherProfileReplyComment}
+          component={withKeyboardWrapper(OtherProfileReplyComment)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="PostDetailPage"
-          component={PostDetailPage}
+          component={withKeyboardWrapper(PostDetailPage)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="ProfilePostDetailPage"
-          component={ProfilePostDetail}
+          component={withKeyboardWrapper(ProfilePostDetail)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="OtherProfilePostDetailPage"
-          component={OtherProfilePostDetail}
+          component={withKeyboardWrapper(OtherProfilePostDetail)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="ChannelScreen"
-          component={ChannelScreen}
+          component={withKeyboardWrapper(ChannelScreen)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="WebsocketResearchScreen"
-          component={WebsocketResearchScreen}
+          component={withKeyboardWrapper(WebsocketResearchScreen)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="SampleChatScreen"
-          component={SampleChatScreen}
+          component={withKeyboardWrapper(SampleChatScreen)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="SampleChatInfoScreen"
-          component={SampleChatInfoScreen}
+          component={withKeyboardWrapper(SampleChatInfoScreen)}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="SignedChatScreen"
-          component={SignedChatScreen}
+          component={withKeyboardWrapper(SignedChatScreen)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here involve wrapping each screen component with the `KeyboardWrapper` component using the `withKeyboardWrapper` function. This is a good practice as it ensures that all screens will have consistent keyboard behavior. However, please ensure that none of these components have their own keyboard handling logic that could potentially conflict with the `KeyboardWrapper`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->